src/gui/sdlmain.cpp

```diff
@@ -2762,17 +2774,22 @@ static void GUI_StartUp(Section *sec)
 			sdl.opengl.npot_textures_supported = gl_version_major >= 2 ||
 			        SDL_GL_ExtensionSupported("GL_ARB_texture_non_power_of_two");
 
+			std::string npot_support_msg = sdl.opengl.npot_textures_supported
+			                                       ? "supported"
+			                                       : "not supported";
+			if (sdl.opengl.npot_textures_supported && !is_shader_flexible()) {
```

Comment:
This would not work when switching shaders at runtime.

Reply:
Good catch @NicknineTheEagle. The wrap parameter (clamp to edge vs. clamp to boundary) falls is in a similar state.

I guess we have move the setting of NPOT and the wrap parameter into the shader loading code - so these are assessed and adjusted on every shader load attempt.

Reply:
Looked at this a bit more.  Turns out the decision to allow window resizing (or not) based on the user's configured shader is also a one-time check during startup, which is similarly affected by this comment.

For now, this code handles the 99%-scenario just like the Window-resize test also handles, so I'm going to merge it as-is.

Addressing this would require some refactoring to avoid repeating code - and because the window-resize test has the same code, it would also affect that too.  A follow on PR in the 0.79 timeframe can refactor all three features affected by problematic shaders:
 - Window resizing
 - Wrap parameter handling
 - NPOT textures